apps/src/feedback.js

```diff
@@ -529,18 +521,10 @@ FeedbackUtils.prototype.displayFeedback = function (
   }
 };
 
-function logDialogActions(event, options, isPerfect) {
-  if (experiments.isEnabled(experiments.FINISH_DIALOG_METRICS)) {
-    firehoseClient.putRecord({
-      study: FIREHOSE_STUDY,
-      study_group: dialog_type,
-      event: event,
-      data_json: JSON.stringify({
-        level_type: options.level ? options.level.skin : null,
-        level_id: options.response ? options.response.level_id : null,
-        level_path: options.response ? options.response.level_path : null,
-        isPerfectBlockCount: isPerfect,
-      }),
+function recordShare(type, appType) {
+  if (Object.prototype.hasOwnProperty.call(EVENTS, type)) {
```

Comment:
nit: is this different than `if EVENTS[type]`?

Reply:
No and `EVENTS[type]` is more simple. Removing condition based on your comment at https://github.com/code-dot-org/code-dot-org/pull/57127#discussion_r1516963716.
But updating the condition in `ShareAllowedDialog`.